plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryLargeImageNextPrevious.php

```diff
@@ -122,7 +122,7 @@ protected function get_button( $button_type ) {
 
 		return sprintf(
 			'<button
-				data-wc-bind--disabled="state.disable%1$s"
+				data-wc-bind--disabled="context.disable%1$s"
```

Comment:
Not a blocker because its not your code, but I really dislike this pattern because its so hard to lookup where values are being used via the IDE search functionality.

Reply:
Great catch! Fixed in https://github.com/woocommerce/woocommerce/pull/55237/commits/03953f9afcc906faaae2421aca1eee9182e1588d